src/PointwiseFunctions/AnalyticData/GrMhd/KomissarovShock.cpp

```diff
@@ -0,0 +1,232 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/GrMhd/KomissarovShock.hpp"
+
+#include <cmath>
+#include <ostream>
+#include <pup.h>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataVector.hpp"  // IWYU pragma: keep
+#include "Parallel/PupStlCpp11.hpp"
+#include "Utilities/ConstantExpressions.hpp"  // IWYU pragma: keep
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/Math.hpp"  // IWYU pragma: keep
+
+namespace {
+template <typename DataType>
+Scalar<DataType> compute_piecewise(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x, const double& inner_value,
```

Comment:
Is there a particular reason for using `inner_value` and `outer_value` as opposed to `left_value` and `right_value`? The current names make me think more of of a box function, which I don't think this is.